### PR TITLE
Commit More Examples

### PR DESCRIPTION
**Pull request by alakae**
Adds examples for:
- Using ingress-nginx
- Using PROXYv2
- Using externalTrafficPolicy "Local"

Feel free to suggest modifications.

### CODE REVIEW

File: examples/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+# Wait for `kubectl describe service ingress-nginx-controller -n ingress-nginx` to
+# show "Loadbalancer Ensured", then use the IP address found under "LoadBalancer
+# Ingress" to connect to the service.
+#
+# You can also use the following shortcut:
+#
+#     curl http://$(kubectl get service ingress-nginx-controller -n ingress-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')/blue
+#     curl http://$(kubectl get service ingress-nginx-controller -n ingress-nginx -o jsonpath='{.status.loadBalancer.ingress[0].ip}')/red
+#
+
```
Comment (href) [inline examples/ingress.yaml:22]: Nit: I would remove this extra space.

File: examples/ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Deploys two nginxdemos/hello:plain-text containers ("blue" and "red")
+# and creates a ClusterIP services named "blue-svc" and "red-svc" for the
```
Comment (href) [inline examples/ingress.yaml:2]:
```suggestion
# and creates ClusterIP services named "blue-svc" and "red-svc" for the
```

File: examples/proxy-v2-protocol.yml [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+          configMap:
+            name: nginx-config
+        - name: hello-plain-config-volume
+          configMap:
+            name: hello-plain-config
+---
+apiVersion: v1
+data:
+  nginx.conf: |2
+
```
Comment (href) [inline examples/proxy-v2-protocol.yml:61]: Nit: I think this space, and a bunch of the double spaces in this config are not needed.

File: examples/external-traffic-policy-local.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+#
+#     kubectl logs -l "app=local-hello"
+#
+# A load balancer health monitor has been configured to check which node a replica is scheduled on.
+# You can use the following script to mimic the check performed by the load balancer health monitor:
+#
+#     HEALTH_CHECK_PORT=$(kubectl get svc local-hello -o jsonpath='{.spec.healthCheckNodePort}')
+#     NODE_IPS=$(kubectl get nodes --selector='!node.kubernetes.io/exclude-from-external-load-balancers' -o jsonpath='{.items[*].status.addresses[1].address}')
+#     for NODE_IP in $NODE_IPS; do curl  -s -o /dev/null -w "%{remote_ip}\t%{http_code}\t%{url_effective}\n" "http://$NODE_IP:$HEALTH_CHECK_PORT/livez"; done
+
```
Comment (href) [inline examples/external-traffic-policy-local.yml:26]: Nit: I would remove those two extra spaces.

**Comment by href** [comment]
I have some nitpicky spacing suggestions, but nothing that needs a second look.

**Review by href — APPROVED** [review]

File: examples/ingress.yaml (hunk 1 above)
Comment (alakae) [reply examples/ingress.yaml]: unified across all examples

File: examples/ingress.yaml (hunk 2 above)
Comment (alakae) [reply examples/ingress.yaml]: fixed.

File: examples/external-traffic-policy-local.yml (hunk 4 above)
Comment (alakae) [reply examples/external-traffic-policy-local.yml]: unified across all examples.

File: examples/proxy-v2-protocol.yml (hunk 3 above)
Comment (alakae) [reply examples/proxy-v2-protocol.yml:62]: I have created this part using `kubectl create configmap --from-file=` using the original nginx config file from the image. Therefore, I prefer to not fiddly around with this part more than necessary. I hope this is okay.

**Pull request merged**